Machine_Learning/ML_For_Sequential_Data_&_Visual_Data/ML_8_14_Single_Shot_Detector_(V)

```diff
@@ -0,0 +1,17 @@
+## Title : Single Shot Detector
+
+🔴 **Type of Content** : Video
+
+🔴 **Domain** : Machine Learning
+
+🔴 **Module** : ML for Seqential Data & Visual Data
+
+*********************************************************************
+
+### Click here for the audio file,

```

Comment:
This will be video file

Reply:
What do I need to do to fix it? @abhisheks008